src/ctia/flows/crud.clj

```diff
@@ -75,34 +74,37 @@
             :entity entity})))
       (id/str->short-id id))))
 
-(defn gen-random-uuid []
-  (UUID/randomUUID))
-
-(defn make-id
-  [entity-type]
-  (str (name entity-type) "-" (gen-random-uuid)))
-
-(s/defn ^:private find-entity-id :- s/Str
```

Comment:
This function used to implicitely consider the type of flow from its content. `(find-id prev-entity)` was only related to update and patch and the other elements to different create cases.
I decomposed it in `find-create-entity-id` and `find-existing-entity-id`.